### PR TITLE
add No Social option story

### PR DESCRIPTION
**Pull request by linroex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@ User stories and acceptance criteria of Uber-like app
 
 * As a user on budget, I want to find a lowest tier car, so that I can save money.
   * Given a nearest parterners list interface, when user choose to prefer cheapest option, then make the sorting mainly driven by lowest tier fist.
+* As a user, I want take a rest without any social and talk.
+  * Option on trip or user: No Social Trip.
```
Comment (xdavidwu) [inline README.md]: try to keep it in given, when, then format?

**Review by xdavidwu — COMMENTED** [review]
a nitpick otherwise lgtm

File: README.md [hunk 2]
```diff
@@ -3,3 +3,5 @@ User stories and acceptance criteria of Uber-like app
 
 * As a user on budget, I want to find a lowest tier car, so that I can save money.
   * Given a nearest parterners list interface, when user choose to prefer cheapest option, then make the sorting mainly driven by lowest tier fist.
+* As a user, I want take a rest without any social and talk.
```
Comment (xdavidwu) [inline README.md]: also add a "so that" clause for this?

**Review by xdavidwu — COMMENTED** [review]
another nitpick

File: README.md (hunk 2 above)
Comment (linroex) [reply README.md]: 但這個場景似乎不會有 so that 的狀況，因為他所獲得的利益其實前一句就是了

File: README.md (hunk 1 above)
Comment (linroex) [reply README.md]: ok, resolved

File: README.md (hunk 2 above)
Comment (xdavidwu) [reply README.md]: `I want minimum conversation, so that I can take a rest`?

Comment (linroex) [reply README.md]: got it, thanks

**Review by xdavidwu — APPROVED** [review]

**Pull request merged**